frontend/src/index.css

```diff
@@ -310,6 +311,19 @@ Other button classes are defined further down together with other classes for th
   color: hsl(185deg 92% 16%);
 }
 
+.login-img {
```

Comment:
Instead of adding a new class you can also nest:
```
.header-wrapper img {
....
```

Reply:
Good idea! :+1: